build-tools/packages/build-infrastructure/src/packageJsonUtils.ts

```diff
@@ -8,8 +8,8 @@ import { readFile } from "node:fs/promises";
 import path from "node:path";
 
 import detectIndent from "detect-indent";
-// Imports are written this way for CJS/ESM compat
 import fsePkg from "fs-extra";
+// eslint-disable-next-line import/no-named-as-default-member -- Imports are written this way for CJS/ESM compat
```

Comment:
Are these just warnings and we're being good citizens and documenting them? Can't be that enabling the other rule somehow made this one start complaining, right?